cms/djangoapps/contentstore/views/item.py

```diff
@@ -266,6 +269,8 @@ def service(self, block, service_name):
                 return DjangoXBlockUserService(self._user)
             if service_name == "studio_user_permissions":
                 return StudioPermissionsService(self._user)
+            if service_name == "lti-configuration":
+                return ConfigurationService(LTIConsumerHideFieldsFlag)
```

Comment:
In this case, my thinking was that the LTI-allow question was a platform level configuration, rather than being course-content. As such, there aren't any xblock fields that make sense. I do agree that if we relaxed the restriction on blocks using django, we'd end up in a clearer situation. We'd also be better off if there were ways xblocks to supply services to the core without requiring a code modification.